.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: java
+
+jdk:
+  - oraclejdk7
+  - oraclejdk8
+
+install: mvn -f gson install -DskipTests=true
+script: mvn -f gson test
```

Comment:
Would love to see the repo updated to a more traditional Java repo layout and some of the other cruft removed or consolidated.